### ISSUE TITLE
systemd unit fails

### ISSUE THREAD

**Issue by FlorianLudwig** [gicmo/bolt#34]
Package used:
```
bolt.x86_64                                0.1-1.fc27                   @gicmo-bolt
```

Error message:
```
systemd[758]: bolt.service: Failed at step NAMESPACE spawning /usr/libexec/boltd: No such file or directory
```
The file does exist though:
```
# ls -alh /usr/libexec/boltd
-rwxr-xr-x. 1 root root 127K 13. Dez 18:32 /usr/libexec/boltd
```

**Comment by gicmo** [comment]
Yep. Mea culpa. Took me a bit on the WE to sort this out. It is because of the sandboxing in the unit file. The `ReadWritePaths` expects the path to exist (unless the have a `-` prefix) and if it doesn't if fails with that not-so-obvious namespace error message. I hopefully fixed with commit 76d349b5fa19ccbf6e6d0418858acb482fb3d358 by creating the directory via an install script. 
The package `0.2-20171217giteb63510.fc27` in the copr should have the fix. I also pushed a (working) package to fedora rawhide and it is [waiting](https://bodhi.fedoraproject.org/updates/FEDORA-2017-0485b38286) for inclusion to f27 too.

NB: I also renamed the directory from `/var/lib/thunderbolt` to `/var/lib/boltd`. So if you already authorized devices, just might wanna move the directory.
